### PR TITLE
override code with state in clickot/ungoogled-chromium-build

### PR DESCRIPTION
**Pull request by clickot**
as agreed in https://github.com/clickot/ungoogled-chromium-build/issues/3 I would override the state of this repo with the state of https://github.com/clickot/ungoogled-chromium-build and would like to get write access this this repo.
Thanks!

### CODE REVIEW

**Comment by clickot** [comment]
@Eloston I don't know how to assign this PR to you, but here it is :)

**Comment by clickot** [comment]
thanks for replying:) I'll wait until @Eloston gives his opinion

File: build.sh [hunk 1]
```diff
@@ -1,46 +1,73 @@
...
+# and GOOGLE_DEFAULT_CLIENT_SECRET at runtime (taken from ungoogled-chromium-archlinux)
+patch -Np1 -i ${patches_dir}/use-oauth2-client-switches-as-default.patch
+# Disable kGlobalMediaControlsCastStartStop by default
+# https://crbug.com/1314342
+patch -Np1 -i ${patches_dir}/disable-GlobalMediaControlsCastStartStop.patch
+# VAAPI wayland support (taken from ungoogled-chromium-archlinux)
+patch -Np1 -i ${patches_dir}/ozone-add-va-api-support-to-wayland.patch
+# fix missing includes in av1_vaapi_video_encoder_delegate.cc
+patch -Np1 -i ${patches_dir}/av1_vaapi_video_encoder_delegate.patch
+
```
Comment (Eloston) [inline build.sh:49]: nit: normally I'd apply patches after ungoogled-chromium ones, but they're unlikely to conflict anyway.

File: docker-build.sh [hunk 2]
```diff
@@ -1,16 +1,38 @@
...
+DISTRO=$(echo ${DISTRO_RELEASE}| cut -d':' -f1)
+RELEASE=$(echo ${DISTRO_RELEASE}| cut -d':' -f2)
+[ "$LLVM_VERSION" -gt "16" ] || REPO_POSTFIX="-$LLVM_VERSION"
+
+IMAGE="chromium-builder-${RELEASE}:llvm-${LLVM_VERSION}"
+
+cd $BASE_DIR 
+
+echo "docker buildx build -t ${IMAGE} --build-arg DISTRO=${DISTRO} --build-arg RELEASE=${RELEASE} --build-arg LLVM_VERSION=${LLVM_VERSION} --build-arg REPO_POSTFIX=${REPO_POSTFIX} --build-arg NODE_VERSION=${NODE_VERSION} ."
+(cd $BASE_DIR/docker && docker buildx build -t ${IMAGE} --build-arg DISTRO=${DISTRO} --build-arg RELEASE=${RELEASE} --build-arg LLVM_VERSION=${LLVM_VERSION} --build-arg REPO_POSTFIX=${REPO_POSTFIX} --build-arg NODE_VERSION=${NODE_VERSION} . )
```
Comment (Eloston) [inline docker-build.sh:20]: nit: can use `set -x` before this line and `set +x` after this line so just this line is printed without needing an explicit `echo` command

It took me a minute to realize you were just echoing the command lol.

**Review by Eloston — APPROVED** [review]
Didn't test but LGTM. I added some nits that you can address and optionally fix.

Learned about `docker buildx` plugin which is pretty neat.

**Comment by Eloston** [comment]
@clickot I invited you with Maintain permission to this repo

@networkException Feel free to change the permissions and/or switch the permissions to teams if you prefer.

**Comment by clickot** [comment]
@Eloston  thank you very much for your invitation as a mainainer! 
I will incorporate your suggestions for improvement before merging

**Comment by networkException** [comment]
> Feel free to change the permissions and/or switch the permissions to teams if you prefer.

I've sent an invite to @clickot, setting manual per user permissions is fine as well though

**Comment by clickot** [comment]
I thank you for your trust and I feel honored :)

**Pull request merged**